joomla/joomla-cms
# Convert mod_articles_categories to new structure

### Summary of Changes
This PR converts mod_articles_categories to the new structure

### Testing Instructions
- Use Joomla 4.3
- Create an instance of the module Articles - Categories. Note how it works.
- Apply patch
- Delete administrator/cache/autoload_psr4.php
- Refresh the page, make sure the module still displays the same as before.

### Additional comments
I not sure of the best approach to use cache here, _if is needed_.
